docs/data/api/menu-positioner.json

```diff
@@ -52,6 +52,22 @@
   "imports": [
     "import { Menu } from '@base-ui-components/react/menu';\nconst MenuPositioner = Menu.Positioner;"
   ],
+  "cssVariables": {
+    "--anchor-height": { "description": "The anchor's height.", "type": "number" },
+    "--anchor-width": { "description": "The anchor's width.", "type": "number" },
+    "--available-height": {
+      "description": "The available height of the positioner element.",
+      "type": "number"
+    },
+    "--available-width": {
+      "description": "The available width of the positioner element.",
+      "type": "number"
+    },
+    "--transform-origin": {
+      "description": "The calculated `transform-origin` CSS property value (to be used on the popup element).",
```

Comment:
I’d say "The coordinates that this element is anchored to. Used for animations and transitions."